### PR TITLE
Improve Themes

### PR DESCRIPTION
**Pull request by onesounds**
## What's the PR
- Removed Duplicated Color scheme Themes 
- Add New Themes / Adjust old themes
- Add Theme Share Page Link
- Adjust Theme Structure
  - Add Bullet, Margin, Selected Glyph Color,  Etc
- Adjust Align when 'Hide Hotkey' setting
- Add Text Trimming (CharacterEllipsis)
- Correct the typo / Fix Clock in Preview Area
- Fix Query Suggestion not display some themes

## Test Cases
- [x] Check Theme Share Page link
- [x] All Themes works fine with clock and date on/off
- [x] 'System' themes works with system color scheme (dark mode)
- [x] Glyph Color should works fine when selected.

### CODE REVIEW

File: Flow.Launcher/Flow.Launcher.csproj [hunk 1]
```diff
@@ -47,7 +47,14 @@
...
   
   <ItemGroup>
+    <Page Remove="Themes\Bullet Light.xaml" />
+    <Page Remove="Themes\Circle Light.xaml" />
+    <Page Remove="Themes\Circle System.xaml" />
+    <Page Remove="Themes\Cyan Dark.xaml" />
+    <Page Remove="Themes\Gray.xaml" />
+    <Page Remove="Themes\Midnight.xaml" />
     <Page Remove="Themes\ThemeBuilder\Template.xaml" />
+    <Page Remove="Themes\Ubuntu.xaml" />
```
Comment (taooceros) [inline Flow.Launcher/Flow.Launcher.csproj:57]: is this change intended?

Comment (onesounds) [reply Flow.Launcher/Flow.Launcher.csproj:57]: noooo. csproj always do it like my kid.

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
@onesounds don't forget to update the Theme customisation doco once 1.10.0 is out if needed.
